File: documentation/user/en/solve/examples/render-category-menu/dynamic-collapsible-menu-sub-category.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	Require(
        		HierarchyOfReference(
        			"categories",
        			RemoveEmpty,
        			FromNode(
        				"dynamicMenuSubcategories",
        				Node(
        					FilterBy(
        						EntityPrimaryKeyInSet(66482)
        					)
        				),
        				EntityFetch(
        					AttributeContent("code")
        				),
        				StopAt(
        					Distance(1)
        				),
        				Statistics(WithoutUserFilter, ChildrenCount)
        			)
        		),
        		Page(1, 0)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/render-category-menu/dynamic-collapsible-menu-sub-category.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				require(
					hierarchyOfReference(
						"categories",
						fromNode(
							"dynamicMenuSubcategories",
							node(
								filterBy(
									entityPrimaryKeyInSet(66482)
								)
							),
							entityFetch(
								attributeContent("code")
							),
							stopAt(
								distance(1)
							),
							statistics(WITHOUT_USER_FILTER, CHILDREN_COUNT)
						)
					),
					page(1, 0)
				)
			)
		);
	}
);
